src/echarts.js

```diff
@@ -1682,6 +1689,14 @@ echartsProto.clear = function () {
         disposedWarning(this.id);
         return;
     }
+
+    // clear all events
+    var handlers = this.getHandlers();
```

Comment:
I think the handler registered by users should better not be cleared when users called `clear`.
It depends on how we define the term `clear`.
But at least, if making change, that will be a break change.

Users call unregister the listeners via calling `chart.off()`.